src/main/java/org/myrobotlab/swing/ProgramABGui.java

```diff
@@ -210,6 +190,15 @@ public void run() {
         progABPath.setText(new File(programab.getPath()).getAbsolutePath());
         userName.setText(programab.getCurrentUserName());
         botName.setText(programab.getCurrentBotName());
+        textAreacontent = "<font face=\"Verdana\" size=-1>";
+        programab.getConversationHistory().stream().forEach(conversation -> {
+          if (conversation == null) {
+            conversation = "ERROR: NULL";
+          }
+          textAreacontent = textAreacontent + conversation + "<br>";
+
+        });
+        response.setText(textAreacontent);

```

Comment:
Does this reconstruct the entire content if a new interaction is made?
I would have expected there to be a more efficient way (like only adding the new lines).
But I'm ok with this if there's no other easy way.